bazel/cargo/wasmtime/Cargo.toml

```diff
@@ -13,30 +13,3 @@ anyhow = "1.0"
 once_cell = "1.12"
 wasmtime = {version = "9.0.3", default-features = false, features = ['cranelift']}
 wasmtime-c-api-macros = {git = "https://github.com/bytecodealliance/wasmtime", tag = "v9.0.3"}
-
-[package.metadata.raze]
-rust_rules_workspace_name = "rules_rust"
-gen_workspace_prefix = "wasmtime"
-genmode = "Remote"
-package_aliases_dir = "."
-workspace_path = "//bazel/cargo/wasmtime"
-
-[package.metadata.raze.crates.cranelift-isle.'*']
-patches = [
-    "@proxy_wasm_cpp_host//bazel/cargo/wasmtime:cranelift-isle.patch",
-]
-patch_args = ["-p4"]
-
-[package.metadata.raze.crates.bumpalo.'*']
-patches = [
-    "@proxy_wasm_cpp_host//bazel/cargo/wasmtime:bumpalo.patch",
-]
-patch_args = ["-p1"]
```

Comment:
If no longer needed, then both patches should be removed from the repository.

Reply:
Yup, done.